org-code-javabuilder/lib/src/main/java/org/code/javabuilder/AWSFileManager.java

```diff
@@ -68,6 +72,16 @@ public String writeToFile(String filename, byte[] inputBytes, String contentType
     }
 
     this.writes++;
+    try {
+      inputStream.close();
```

Comment:
So I think we don't actually need to worry about this for ByteArrayInputStreams? According to the docs closing has no effect: https://docs.oracle.com/javase/7/docs/api/java/io/ByteArrayInputStream.html#close()

Reply:
oh good call, I'll remove this